analyze-slow-queries.md

```diff
@@ -0,0 +1,255 @@
+---
+title: Analyze Slow Queries
+summary: Learn how to locate and analyze slow queries.
+---
+
+# Analyze Slow Queries
+
+To address the issue of slow queries, you need to take the following two steps:
+
+1. Among many queries, identify which type of queries are slow.
+2. Analyze why this type of queries are slow.
+
+You can easily perform step 1 using the [slow query log](/identify-slow-queries.md) and the [statement summary table](/statement-summary-tables.md) features. It is recommended to use TiDB Dashboard, which integrates the two features and directly displays the slow queries in your browser. 
```

Comment:
It would be better if we point to the slow query page on TiDB Dashboard: https://docs.pingcap.com/tidb/dev/dashboard-slow-query

Reply:
Thanks! Updated in https://github.com/pingcap/docs/pull/4035/commits/6f7fd5c7bd076186f18c9a9cdfd3df4667f60301.